coalib/settings/ConfigurationGathering.py

```diff
@@ -490,11 +493,13 @@ def gather_configuration(acquire_settings,
                                            args=args)
     _set_section_language(sections)
     aspectize_sections(sections)
-    local_bears, global_bears = fill_settings(sections,
-                                              acquire_settings)
+    local_bears, global_bears = fill_settings(sections, acquire_settings)
     save_sections(sections)
     warn_nonexistent_targets(targets, sections)
+    parsed_settings = get_custom_settings(arg_list, arg_parser, args)
```

Comment:
why do we need parsed settings?

They have already been parsed once?

I thought they are put into the `cli` section?